src/Microsoft.AspNet.Razor/Parser/HtmlMarkupParser.Block.cs

```diff
@@ -558,19 +558,37 @@ private void AttributeValue(HtmlSymbolType quote)
 
             if (At(HtmlSymbolType.Transition))
             {
-                var valueStart = CurrentLocation;
-                PutCurrentBack();
+                if (NextIs(HtmlSymbolType.Transition))
+                {
+                    // Render a single "@" in place of "@@".
+                    using (Context.StartBlock(BlockType.Markup))
+                    {
+                        Span.CodeGenerator = new LiteralAttributeCodeGenerator(
+                            prefix.GetContent(prefixStart),
+                            new LocationTagged<string>(CurrentSymbol.GetContent(), CurrentLocation));
```

Comment:
Does it matter what code generator is used for the `prefix`?  More to the point, any downside of using `LiteralAttributeCodeGenerator` for that (in addition to the literal `@`?